vignettes/baggr.Rmd

```diff
@@ -294,24 +296,22 @@ forest_plot(baggr_schools, show = "both")
 
 ## Cross-validation in _baggr_
 
-_Baggr_ has built-in, automated leave-one-out cross validation for its models, where the "one" always refers to one group. This naturally corresponds to the question of how much any one group has contributed to the overall inference. Therefore, if you have $K$ groups, using the `loocv()` will run your model of choice $K$ times. Be aware that this may take a while even for simple models. (You should see a progress bar in your terminal window.) The `loocv()` function takes in the same arguments as `baggr()`, plus an option (`return_models`) for whether to return all the models or just the summary statistics. For the 8 schools example we can do
+_Baggr_ has built-in, automated leave-one-out cross validation for its models. The values returned by `loocv()` can be used to understand how any
+one group affects the overall result, as well as how well the model predicts the omitted group.
 
-```{r loocv, eval = FALSE}
-loocv_res <- loocv(schools, return_models = FALSE, "rubin", pooling = "partial")
-```
+This function automatically runs $K$ baggr models, leaving out one group at a time, and then calculates expected log predictive density (ELPD) for that group (see Gelman et al 2013). The main output is the cross-validation information criterion, or -2 times the ELPD averaged over $K$ models. This is related to, and often approximated by, the Watanabe-Akaike Information Criterion. A value closer to zero (i.e. a smaller number in magnitude) means a better fit. For more information on cross-validation see this [overview article](http://www.stat.columbia.edu/~gelman/research/published/waic_understand3.pdf).
```

Comment:
For .Rmd's we use a .bib file (you can't easily do it for .Rd, unfortunately...), so here you can directly do `@gelman_understanding_2014` and then add this to bib file:

```
@article{gelman_understanding_2014,
  title = {Understanding Predictive Information Criteria for {{Bayesian}} Models},
  volume = {24},
  issn = {0960-3174, 1573-1375},
  abstract = {We review the Akaike, deviance, and Watanabe-Akaike information criteria from a Bayesian perspective, where the goal is to estimate expected out-of-sample-prediction error using a biascorrected adjustment of within-sample error. We focus on the choices involved in setting up these measures, and we compare them in three simple examples, one theoretical and two applied. The contribution of this review is to put all these information criteria into a Bayesian predictive context and to better understand, through small examples, how these methods can apply in practice.},
  language = {en},
  number = {6},
  journal = {Statistics and Computing},
  doi = {10.1007/s11222-013-9416-2},
  author = {Gelman, Andrew and Hwang, Jessica and Vehtari, Aki},
  month = nov,
  year = {2014},
  pages = {997-1016},
  file = {C\:\\Users\\wwiecek.CERTARA\\Zotero\\storage\\KDSC9Z5I\\Gelman et al. - 2014 - Understanding predictive information criteria for .pdf},
  note = {00642}
}
```

Reply:
Yup, makes sense!

Reply:
Updated this language and adopted this usage. Hopefully I didn't miss any additional references.